CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+### 2.20.0
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.8.7](https://fluentbit.io/announcements/v1.8.7/)
+* Amazon CloudWatch Logs for Fluent Bit 1.6.3
+* Amazon Kinesis Streams for Fluent Bit 1.8.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.19.2` this release adds:
+* Feature - Add support for gzip compression of records [kinesis:162](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit/pull/162)
+* Bug - Fix return value from `tls_net_read` [fluentbit:4100](https://github.com/fluent/fluent-bit/pull/4100)
+* Bug - Downgrade `mbedtls` to 2.24.0 [firelens:249](https://github.com/aws/aws-for-fluent-bit/pull/249)
```

Comment:
> [firelens:249]

firelens isn't the repo name... `aws-for-fluent-bit` would make more sense

Reply:
also, I think we should add something saying why we did this? Like say its because it improves performance. Ramya can help with the wording

Reply:
For mbedtls, link to the github issue https://github.com/fluent/fluent-bit/issues/4110. Wording will be: `Revert to mbedtls 2.24.0 because mbedtls 2.26.0 introduces performance regression in the function used to encode the request to AWS destinations in base64 format.`

Reply:
Sure! Let me update to this